app/src/common/shared/org/mozilla/vrbrowser/search/SearchEngine.java

```diff
@@ -83,6 +83,8 @@ private SearchEngine(@NonNull Context aContext) {
         String geolocationJson = SettingsStore.getInstance(mContext).getGeolocationData();
         GeolocationTask.GeolocationData data = GeolocationTask.GeolocationData.parse(geolocationJson);
         mEngine = Engine.getEngine(data);
+
+        SessionStore.get().setRegion(data.getCountryCode());
```

Comment:
I think we should also set the region in onGeolocationRequestSuccess so we always get the most updated region after a geolocation request finishes.

Reply:
oh, good call